src/js/components/put-away/SplitLineModal.jsx

```diff
@@ -202,7 +201,22 @@ class SplitLineModal extends Component {
           shouldCloseOnOverlayClick={false}
         >
           <div>
-            <h3 className="font-weight-bold">{`${this.props.putawayItem.product.productCode} ${this.props.putawayItem.product.displayNameOrDefaultName}`}</h3>
+            <h3 className="font-weight-bold">
+              <span>{this.props.putawayItem.product.productCode}</span>
+              <Tooltip
+                arrow="true"
+                delay="150"
+                duration="250"
+                hideDelay="50"
+                className="text-overflow-ellipsis"
+                disabled={this.props.putawayItem.product?.name ==
+                  this.props.putawayItem.product?.displayNameOrDefaultName}
```

Comment:
1. why `==` instead of `===`
2. can't we just check if `disabled={!this.props.putawayItem.product?.displayNameOrDefaultName}`

Reply:
double equals instead of a single is just a typo, but as for `disabled={!this.props.putawayItem.product?.displayNameOrDefaultName}`
displayNameOrDefaultName is always populated, jus tlike the name suggests it returns displayName when it is present or default name

Reply:
oh, I'm sorry, I have not noticed you used the `displayNameOrDefaultName`.